utils/customize.py

```diff
@@ -0,0 +1,81 @@
+import json
+from pathlib import Path
+
+try:
+	import frappe
+	from frappe.database.schema import add_column
+	from frappe.cache_manager import clear_global_cache
+	from frappe.modules.import_file import calculate_hash
+except Exception as e:
+	raise(e)
+
+
+def load_customizations():
+	apps = frappe.get_installed_apps()
+	frappe_apps = ["frappe", "erpnext", "hrms", "payments", "lms", "insights"]
+	for app in apps:
+		if app in frappe_apps:
+			continue
+
+		print(f"Loading {app} customizations")
+		customizations_directory = Path().cwd().parent / "apps" / app / app / app / "custom"
```

Comment:
I think this code isn't correct in all cases and I want to make sure it is before we release. If we have an app named "xyz" and the only module in that app is named "abc", we won't find the customizations. So I think we need to iterate both apps AND modules in those apps.

Reply:
Okay, will iterate over modules as well.